### ISSUE TITLE
Containerize monolithic-layered app

### ISSUE THREAD

**Issue by naXa777** [aabarmin/epam-microservices-training-2022#31]
> Docker helps you containerize your Java app — letting you bundle together your complete Spring application, runtime, configuration, and OS-level dependencies. This includes everything needed to ship a cross-platform, multi-architecture web application.

Plan:
1. Create Dockerfile
    * This can be done by following the guide: [Kickstart Your Spring Boot Application Development](https://www.docker.com/blog/kickstart-your-spring-boot-application-development)
2. Create docker-compose file

**Comment by youngInnovator** [comment]
Kindly assign this to me.

**Comment by aabarmin** [comment]
Hi @youngInnovator 
Assigned, thank you for contributing

**Comment by youngInnovator** [comment]
Addressed in #46

**Comment by aabarmin** [comment]
This looks like a good option, I am slightly worried that this approach will download dependencies every time the container is created from the image. What do you think, would it be better if monolithic apps are built using the same approach as all other microservices - using `docker/build.sh` script? 

The script is already available here - https://github.com/aabarmin/epam-microservices-training-2022/blob/main/buildscripts/build.sh

To add support of a new application, it is necessary to add a line like: 

```shell
docker build \
    --build-arg module_path=monolithic/monolithic-layered \
    -t monolithic/monolithic-layered:latest \
    -f ./Dockerfile ..
```

This script will automatically build a `jar` artifact from the Spring Boot application and next generate a Docker image.

**Comment by aabarmin** [comment]
Resolved in #46
